nixos/modules/rename.nix

```diff
@@ -134,6 +134,7 @@ with lib;
     (mkRenamedOptionModule [ "services" "piwik" "enable" ] [ "services" "matomo" "enable" ])
     (mkRenamedOptionModule [ "services" "piwik" "webServerUser" ] [ "services" "matomo" "webServerUser" ])
     (mkRenamedOptionModule [ "services" "piwik" "phpfpmProcessManagerConfig" ] [ "services" "matomo" "phpfpmProcessManagerConfig" ])
```

Comment:
Out of curiosity what happens if you have `services.piwik.phpfpmProcessManagerConfig` set in your `configuration.nix` with this branch? (ie. an option which has been renamed and then removed)

Reply:
Yeah I'm not sure if the way I did that is the right way and I don't know this well enough to even know of what other possible approaches there might be. it seemed logical to me, first attempt that fixed the error, seemed to work, and I didn't explore it much after that.

To answer your question...

```
# with services.piwik.*
00:54:19 [kiwi@absurd-nixos nixpkgs]$ sudo nixos-rebuild -I nixpkgs=. build
building Nix...
building the system configuration...
trace: warning: The option definition `services.matomo.phpfpmProcessManagerConfig' in `/home/kiwi/nixpkgs/nixos/modules/rename.nix' no longer has any effect; please remove it.
Use services.phpfpm.pools.<name>.settings
trace: warning: The option `services.piwik.phpfpmProcessManagerConfig' defined in `/etc/nixos/modules/matomo.nix' has been renamed to `services.matomo.phpfpmProcessManagerConfig'.
00:54:34 [kiwi@absurd-nixos nixpkgs]$ sudo vim /etc/nixos/modules/matomo.nix
# changed to services.matomo.*
00:54:58 [kiwi@absurd-nixos nixpkgs]$ sudo nixos-rebuild -I nixpkgs=. build
building Nix...
building the system configuration...
trace: warning: The option definition `services.matomo.phpfpmProcessManagerConfig' in `/etc/nixos/modules/matomo.nix' no longer has any effect; please remove it.
Use services.phpfpm.pools.<name>.settings
00:55:06 [kiwi@absurd-nixos nixpkgs]$
```

Reply:
In that case the message seems confusing so let's modify the `mkRenamedOptionModule` to something like
```
 	    (mkRemovedOptionModule [ "services" "piwik" "phpfpmProcessManagerConfig" ] "Use services.phpfpm.pools.<name>.settings")
```